Docs/sphinx_documentation/source/LinearSolvers.rst

```diff
@@ -925,7 +925,7 @@ viscous term `divtau` explicitly:
        {
            BoxArray edge_ba = grids[lev];
            edge_ba.surroundingNodes(dir);
-           b[lev][dir].reset(new MultiFab(edge_ba, dmap[lev], 1, nghost, MFInfo(), *ebfactory[lev]));
+           b[lev][dir] = std::make_unique<MultiFab>(edge_ba, dmap[lev], 1, nghost, MFInfo(), *ebfactory[lev]);
```

Comment:
@WeiqunZhang There is just one detail here that hit me recently:

if you assign a new `make_unique` data structure that is *large* then there is a short moment in which both the old one and the new one are alive and occupy memory: https://godbolt.org/z/PTr4MsPY8

```C++
#include <iostream>
#include <memory>

struct T {
    T(){ std::cout << "alive\n";}
    ~T(){ std::cout << "dead\n";}
};

int main()
{
    auto p = std::make_unique<T>();
    p = std::make_unique<T>();
}
```
```
alive
alive
dead
dead
```

In case your MultiFabs are large, this might be quite expensive for peak memory.

Reply:
That is true.